opentelemetry-sdk/src/opentelemetry/sdk/_metrics/instrument.py

```diff
@@ -30,14 +31,28 @@
     ObservableUpDownCounter as APIObservableUpDownCounter,
 )
 from opentelemetry._metrics.instrument import UpDownCounter as APIUpDownCounter
+from opentelemetry.sdk._metrics.aggregation import (
+    Aggregation,
+    ExplicitBucketHistogramAggregation,
+    LastValueAggregation,
+    SumAggregation,
+)
 from opentelemetry.sdk._metrics.measurement import Measurement
 from opentelemetry.sdk._metrics.measurement_consumer import MeasurementConsumer
+from opentelemetry.sdk._metrics.point import AggregationTemporality
 from opentelemetry.sdk.util.instrumentation import InstrumentationInfo
 
 _logger = logging.getLogger(__name__)
 
 
-class _Synchronous:
+class _Instrument(ABC):
+    @property
+    @abstractmethod
+    def aggregation(self) -> Aggregation:
+        pass
```

Comment:
I am against adding this to the public API. Why would a user of instrument need this? It would be easy enough to do this inside of the SDK.

There was previously a spec discussion about allowing users to override the default aggregation for each instrument type globally. It would be much easier to implement that outside of the instrument

Reply:
Yes, a user would not need this, I have made it private. :+1: 

I suggest we revisit this implementation if the mentioned discussion leads to a change in the spec.